packages/rest/src/validation/request-body.validator.ts

```diff
@@ -93,9 +92,11 @@ const compiledSchemaCache = new WeakMap();
 function validateValueAgainstJsonSchema(
```

Comment:
Since this function is accepting OpenAPI Schema now, its name needs to be updated too.